docs/testingplatform/Source/TestingPlatformExplorer/TestingFramework/TestingFramework.Services.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text;
+
+namespace TestingPlatformExplorer.TestingFramework;
+
+public interface ITestOutputHelper
+{
+    void WriteLine(string message);
+    void WriteErrorLine(string message);
+}
+
+internal sealed class TestOutputHelper : ITestOutputHelper
+{
+    public StringBuilder Output { get; set; } = new StringBuilder();
+    public StringBuilder Error { get; set; } = new StringBuilder();
```

Comment:
Does it need to be settable?

Reply:
Nope, it's a sample code and the class is internal we remove the set I can do some cleanup as follow-up